calculate_trigger.py

```diff
@@ -133,8 +135,9 @@ def search_for_fault_location(
                     trigger_position - goldenrun_tb_info.at[idtbinfo, "ins_count"]
                 )
             else:
+                tb_id = goldenrun_tb_info.at[idtbinfo, "id"]
                 for filt in filter_lists:
-                    if filt[0] == goldenrun_tb_info.at[idtbinfo, "id"]:
+                    if filt[0] == tb_id:
                         ins = filt[len(filt) - trigger_position]
                         trigger_position = 0
                         break
```

Comment:
It was not written by you, but could we change the lines to to continue if filt does not match the tb_id? For example:

```
if filt[0] != tb_id:
  continue
ins = ...
trigger_position = 0
break
```

Reply:
Should we do the same for the lines 148 and 150?

Reply:
Yes, that would be nice :)